### PR TITLE
[DOC] Fix rendering of plots in some examples where titles or images are cut off

### PR DESCRIPTION
**Pull request by ymzayek**
Handles part of #3728

### CODE REVIEW

**Comment by ymzayek** [comment]
The best way to review this is the doc build artifacts. Here is a link to the examples: https://output.circle-artifacts.com/output/job/caf24ad1-f374-48bc-9e74-1f3d185d7fbc/artifacts/0/dev/auto_examples/index.html
The ones where a thumbnail from an example is rendered are the ones that are affected by this PR.

Note that this PR does not correct the overlapping titles of [matrix plots.](https://nilearn.github.io/dev/auto_examples/03_connectivity/plot_signal_extraction.html#) I will do one more PR to fix those.

File: examples/01_plotting/plot_demo_more_plotting.py [hunk 1]
```diff
@@ -43,6 +43,8 @@
 # First, we retrieve data from nilearn provided (general-purpose) datasets
 # ------------------------------------------------------------------------
 
+import matplotlib.pyplot as plt
```
Comment (Remi-Gau) [inline examples/01_plotting/plot_demo_more_plotting.py:46]: out of curiosity: why is this new import needed?

File: examples/01_plotting/plot_demo_more_plotting.py [hunk 2]
```diff
@@ -127,7 +130,8 @@
 
 plotting.plot_stat_map(stat_img, display_mode='z',
                        cut_coords=1, colorbar=False,
-                       title="display_mode='z', cut_coords=1, colorbar=False")
+                       title="display_mode='z', cut_coords=1, colorbar=False",
```
Comment (Remi-Gau) [inline examples/01_plotting/plot_demo_more_plotting.py:133]: Does it work if you put the title over 2 lines?

```suggestion
                       title="display_mode='z', cut_coords=1,\ncolorbar=False",
```

image

**Review by Remi-Gau — APPROVED** [review]
Good with me !! 🚀

File: examples/01_plotting/plot_demo_more_plotting.py (hunk 1 above)
Comment (ymzayek) [reply examples/01_plotting/plot_demo_more_plotting.py:46]: it's used to change the figure size in the modified plots in L122 and L134. Maybe better to move it below to where it's used

**Review by bthirion — APPROVED** [review]
LGTM, thx.

**Pull request merged**